dev-support/Jenkinsfile

```diff
@@ -131,9 +131,8 @@ curl -L  -o personality.sh "${env.PROJECT_PERSONALITY}"
             // TODO does hadoopcheck need to be jdk specific?
             // Should be things that work with multijdk
             TESTS = 'all,-unit,-findbugs'
-            // on branches that don't support jdk7, this will already be JAVA_HOME, so we'll end up not
-            // doing multijdk there.
-            MULTIJDK = '/usr/lib/jvm/java-8-openjdk-amd64'
+            // this is only elevant on branches-1. on branches with default jdk8, the relevant jdk will be JAVA_HOME
+            MULTIJDK = '/usr/lib/jvm/zulu-8-amd64'
```

Comment:
This still trips me up. I'd rather see both (all) JDK's explicitly listed out in the `--multijdkdirs` argument. It's explicitly clear, vs. "use this one plus whatever I happens to resolve into `JAVA_HOME` later."

Reply:
That's an easy enough change. Will do.